### ISSUE TITLE
Variable reference 'binary /../' is unbound at api-generator-sifive/program/program.wake:282:[5-8]

### ISSUE THREAD

**Issue by saurabhsingh-pvips** [sifive/api-generator-sifive#27]
Updating block-pio-sifive to latest commits results into the following error:

```
Variable reference 'binary /../' is unbound at api-generator-sifive/program/program.wake:282:[5-8]
>>> Aborting without execution <<<
```

**Comment by jackkoenig** [comment]
Are you using wake `0.17.1`?

**Comment by saurabhsingh-pvips** [comment]
I was using `v0.15.1` . Updated wake to `v0.17.1`. The error changes to: 

```
Variable reference 'getPlanHash' is unbound at ../stuff/share/wake/lib/system/incremental.wake:58:[21-31]
Primitive reference shell_str is unbound at ../stuff/share/wake/lib/system/incremental.wake:61:[18-33]
```

**Comment by jackkoenig** [comment]
That is a very strange error. Those functions are defined on `master`, but not on `v0.17.1`. It's like you have `v0.17.1` but are somehow including the standard library from `master`. 

@terpstra is it possible to override Wake's search path for the standard library?

@saurabhsingh-pvips can you try rebuilding wake from a pristine environment?

**Comment by jackkoenig** [comment]
Did you build wake on `v0.17.1` then checkout a different commit in the repo? That might explain it.

**Comment by saurabhsingh-pvips** [comment]
No. I checked out the latest commit for `block-pio-sifive.` Updated wake to `v0.17.1 `from `v0.15.0`. Also removed ./`build `, `build `& `wake.db`. Initialized the `wake.db` again but the error persists.

**UPDATE:**
Fixed it by removing directory where wake was installed. You're correct. I had files other than what `v0.17.1 `has. Checked out `v0.17.1 `and installed. The errors are squashed.
